chef_master/source/chef_license_accept.rst

```diff
@@ -0,0 +1,211 @@
+Users can set the `CHEF_LICENSE` environment variable or add `chef_license: accept` to their provisioner config in their
+`kitchen.yml`. Specifying `accept` will attempt persist the license acceptance locally. If a local license marker file
+is detected, no configuration is required - acceptance is automatically transferred to the test instance.
+
+To disable this persistence specify `accept-no-persist` on every test instance converge.
+
+`kitchen-inspec` uses InSpec as a library and is not covered by the EULA when installed as a gem, but is covered by the
+EULA when packaged as part of the ChefDK installation. Accept the license in a similar way to the Chef Infra license -
+specify the `CHEF_LICENSE` environment variable, specify the `chef_license` config under the verifier section in
+`kitchen.yml` or persist the acceptance locally.
+
+Pin to Chef 14
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+You can pin to a specific version of chef in your kitchen.yml:
+
+.. code-block:: none
+
+  provisioner:
+    name: chef_zero
+    product_name: chef
+    product_version: 14.12.3
+
+`knife bootstrap`
+-----------------------------------------------------
+`knife bootstrap` requires users to accept the Chef Infra license locally before bootstrapping a remote instance. In
+most usage cases (via Chef Workstation) this license will already have been accepted and will transfer across
+transparently. But if a user installs Chef Workstation and the first command they ever run is `knife bootstrap` it will
+perform the same license acceptance flow as the Chef Infra product.
+
+Pin to Chef 14
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Specify the following argument:
+
+.. code-block:: none
+
+  knife bootstrap --bootstrap-version 14.12.3
+
+Vagrant
+-----------------------------------------------------
+This can be done via the arguments API:
+
+.. code-block:: none
+
+   config.vm.provision "chef_zero" do |chef|
+     chef.arguments = "--chef-license accept"
+     ...
+   end
+
+
+See `<https://www.vagrantup.com/docs/provisioning/chef_common.html#arguments>`__ for details. This argument is not backwards compatible so instead users can use the `custom config path <https://www.vagrantup.com/docs/provisioning/chef_common.html#custom_config_path>`__ and point at a local file, which specifies the `chef_license` config. The environment variable is not currently supported.
+
+
+Pin to Chef 14
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+This can be done via the version API:
+
+.. code-block:: none
+
+   config.vm.provision "chef_zero" do |chef|
+     chef.version = "14.12.3"
+     ...
+   end
+
+See `<https://www.vagrantup.com/docs/provisioning/chef_common.html#version>`__
+
+Chef Terraform Provisioner
+-----------------------------------------------------
+Documentation coming soon.
+
+Pin to Chef 14
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+In your provisioner config:
```

Comment:
Is this `provisioner config` a certain file or file type?

Reply:
Changed it to `Terraform provisioner config` to hopefully clarify that it is both a type and a specific file. Terraform uses their own custom syntax.